oscar_elasticsearch/search/settings.py

```diff
@@ -135,4 +136,6 @@
 
 FACET_BUCKET_SIZE = getattr(settings, "OSCAR_ELASTICSEARCH_FACET_BUCKET_SIZE", 10)
 
-INDEXING_CHUNK_SIZE = getattr(settings, "OSCAR_ELASTICSEARCH_INDEXING_CHUNK_SIZE", 100)
+INDEXING_CHUNK_SIZE = lazy(
+    lambda: getattr(settings, "OSCAR_ELASTICSEARCH_INDEXING_CHUNK_SIZE", 100), int
```

Comment:
waarom must this lazy then?

Reply:
@override_settings werkt anders niet in de tests